_posts/2016-12-02-contributing-to-pulp-blog.md

```diff
@@ -0,0 +1,40 @@
+1) Fork [pulpproject.org respository](https://github.com/pulp/pulpproject.org/) on GitHub.
```

Comment:
s/Fork/Fork the/
Also can the url be written out so the user can read it at the end of the line and that become the linkable part?